dan_layer/epoch_manager/src/base_layer/base_layer_epoch_manager.rs

```diff
@@ -36,16 +49,16 @@ use tari_dan_common_types::{
     SubstateAddress,
 };
 use tari_dan_storage::global::{models::ValidatorNode, DbBaseLayerBlockInfo, DbEpoch, GlobalDb, MetadataKey};
-use tari_dan_storage_sqlite::global::SqliteGlobalDbAdapter;
+use tari_dan_storage_sqlite::{error::SqliteStorageError, global::SqliteGlobalDbAdapter};
 use tari_utilities::{byte_array::ByteArray, hex::Hex};
-use tokio::sync::{broadcast, oneshot};
+use tokio::sync::{broadcast, oneshot, Mutex};
 
 use crate::{base_layer::config::EpochManagerConfig, error::EpochManagerError, EpochManagerEvent};
 
 const LOG_TARGET: &str = "tari::dan::epoch_manager::base_layer";
 
 pub struct BaseLayerEpochManager<TGlobalStore, TBaseNodeClient> {
-    global_db: GlobalDb<TGlobalStore>,
+    global_db: Arc<GlobalDb<TGlobalStore>>,
```

Comment:
Global db already has an inner Arc - what is the reason for using an outer Arc?

Reply:
its a leftover, just did a cleanup